bin/eks-create-cluster.sh

```diff
@@ -38,24 +38,24 @@ echo ""
 
 CLUSTER_ARN=$(aws eks create-cluster --name $EKS_CLUSTER_NAME \
               --role-arn $EKS_ROLE_ARN \
-              --resources-vpc-config subnetIds=$EKS_SUBNETS,securityGroupIds=$EKS_SECURITY_GROUPS \
+              --resources-vpc-config subnetIds=$EKS_SUBNETS,securityGroupIds=$EC2_SECURITY_GROUP \
               --query cluster.arn --output text)
 
 echo "EKS Cluster created, usually takes 10 minutes..."
```

Comment:
May be "EKS Cluster **is being** created, usually takes 10 minutes..."

Reply:
If the message appears after creation, then "usually takes 10 minutes ..." may not be required.
If you want to alert the user about the time it takes to create the cluster, then it may be good to make that portion appear before running the create-cluster command.